src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final Properties DB_PROPERTIES = new Properties();
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "root";
+    private static final String DRIVER_NAME = "com.mysql.cj.jdbc.Driver";
+    private static final String URL
+            = "jdbc:mysql://localhost:3306/jdbc_for_mate?serverTimezone=UTC";
```

Comment:
```suggestion
            = "jdbc:mysql://localhost:3306/taxi_service?serverTimezone=UTC";
```
лучше с самого начала переименовать на нейтральное название, потом будем этот код переносить в свой личный проект